pkgs/applications/graphics/pdfcpu/default.nix

```diff
@@ -2,16 +2,36 @@
 
 buildGoModule rec {
   pname = "pdfcpu";
-  version = "0.5.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "pdfcpu";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-dEAlOKjNXL7zqlll6lqGmbopjdplDR3ewMMNu9TMsmw=";
+    hash = "sha256-410DIdsPXKMxnBL+1KX/DIPktqoB/ZxgF5GXEiqLcbw=";
+    leaveDotGit = true;
+    postFetch = ''
+      cd "$out"
+      git rev-parse HEAD > $out/COMMIT
+      date -u -d "@$(git log -1 --pretty=%ct)" "+%Y-%m-%dT%H:%M:%SZ" > $out/SOURCE_DATE_EPOCH
+      find "$out" -name .git -print0 | xargs -0 rm -rf
+    '';
   };
 
-  vendorHash = "sha256-WZsm2wiKedMP0miwnzhnSrF7Qw+jqd8dnpcehlsdMCA=";
+  vendorHash = "sha256-jVX/CFf9dd9qD3gyGVDjbfINtPLCsgdjWFix4BCpDZ0=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=v${version}"
+  ];
+
+  # ldflags based on metadata from git and source
+  preBuild = ''
+    ldflags+=" -X main.commit=$(cat COMMIT)"
+    ldflags+=" -X main.date=$(cat SOURCE_DATE_EPOCH)"
+  '';
+
 
   # No tests
   doCheck = false;
```

Comment:
Maybe we can make the `installCheckPhase` also grep `$(cat COMMIT)` and `$(SOURCE_DATE)`?

Reply:
For me the "Add suggstion to batch" button is disabled. I'm happy to accept all your suggestions if you want to commit them through GitHub. I'm also happy to add them manually and credit you using `Co-authored-by: @doronbehar`.

Any preference?

Reply:
Manually commiting the changes and using `Co-authored-by:` is great.

Reply:
Hopefully I've correctly update this PR with all the suggested and requested changes. Mind having another look, @doronbehar?